cmake/sysbuild/modules/ncs_sysbuild_extensions.cmake

```diff
@@ -39,10 +41,12 @@ function(ExternalNcsVariantProject_Add)
   endif()
 
   set_property(TARGET ${VBUILD_VARIANT} PROPERTY NCS_VARIANT_APPLICATION ${VBUILD_APPLICATION})
-  set_property(TARGET ${VBUILD_VARIANT} APPEND PROPERTY _EP_CMAKE_ARGS
-    -DCONFIG_NCS_IS_VARIANT_IMAGE=y
-    -DPRELOAD_BINARY_DIR=${${VBUILD_APPLICATION}_BINARY_DIR}
-  )
+  if(NOT DEFINED VBUILD_SPLIT_KCONFIG)
+    set_property(TARGET ${VBUILD_VARIANT} APPEND PROPERTY _EP_CMAKE_ARGS
+      -DCONFIG_NCS_IS_VARIANT_IMAGE=y
```

Comment:
`      -DCONFIG_NCS_IS_VARIANT_IMAGE=y`
should be in both

Reply:
If that's what you want - than I need to introduce a dedicated symbol to skip the Kconfig part

Reply:
Hmm see what you mean, add one for the other but give it a different name like `NCS_IS_SPLIT_KCONFIG_VARIANT_IMAGE`

Reply:
So far - I proposed the `CONFIG_NCS_VARIANT_MERGE_KCONFIG` symbol, not exported to the Kconfig.
IMO it should either:
 - not use `CONFIG_` prefix
 - be appended the same way as the `CONFIG_NCS_IS_VARIANT_IMAGE`

Please let me know if you find any value in having this in Kconfig, so I can adjust accordingly.